### PR TITLE
chore: remove flag from new dashboard modal

### PR DESCRIPTION
**Pull request by lharries**
## Changes

Removes the feature flag for the old new dashboard logic

## How did you test this code?

Ran it locally

### CODE REVIEW

**Review by pauldambra — APPROVED** [review]
nice 🙌 

Didn't run it 🙈 but looks correct

**Comment by lharries** [comment]
@pauldambra, the cypress tests failed with the new UI

I've updated them and in order to get the dashboard template test working I added the OG dashboard template as a migration

When/if the tests pass it would be great to get another review

**Comment by lharries** [comment]
Tests seem to be passing

**Comment by lharries** [comment]
Actually need to fix the remaining test briefly

**Comment by lharries** [comment]
@pauldambra should be ready for a re-review now, if you wouldn't mind doubling checking the migration makes sense as I haven't added data to PostHog like this before

**Comment by lharries** [comment]
And do I need to add a reverse step e.g. the Noop or is that optional?

**Comment by lharries** [comment]
Seems like I've lost a commit which fixed the cypress tests, will add it back

**Comment by lharries** [comment]
@pauldambra should be good to go :)

File: posthog/migrations/0309_add_starter_dashboard_template.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0308_add_indirect_person_override_constraints"),
+    ]
+
+    operations = [migrations.RunPython(create_starter_template)]
```
Comment (pauldambra) [inline posthog/migrations/0309_add_starter_dashboard_template.py:156]:
```suggestion
    operations = [migrations.RunPython(create_starter_template, reverse_code=migrations.RunPython.noop)]
```

RunPython throws when reversed if `reverse_code` is not provided. We _should_ be fine to have a no-op reversal instead

**Review by pauldambra — APPROVED** [review]
🚢

**Pull request merged**